src/CBT.NuGet/Tasks/GenerateNugetProperties.cs

```diff
@@ -115,6 +120,7 @@ public bool Execute(string packageRestoreFile, string[] inputs, string propsFile
             }
             catch (Exception e)
             {
+                Log.LogError(e.ToString());
                 Trace.TraceError(e.ToString());
```

Comment:
I was using tracing to be unobtrusive.  You can edit the `MSBuild.exe.config` to get the traces.  Do you think we should be logging unhandled exceptions?